appoptics-agent-deployment.yaml

```diff
@@ -18,13 +18,29 @@ spec:
       serviceAccountName: appoptics-agent-serviceaccount
       containers:
         - name: appoptics-agent-k8s
-          image: 'appoptics/appoptics-agent-docker:v0.6'
+          image: 'appoptics/appoptics-agent-docker:v0.7'
           imagePullPolicy: Always
           env:
             - name: APPOPTICS_TOKEN
               value: 'APPOPTICS_TOKEN'
-            - name: APPOPTICS_ENABLE_KUBERNETES
+            - name: APPOPTICS_ENABLE_APACHE
+              value: 'false'
+            - name: APPOPTICS_ENABLE_DOCKER
               value: 'true'
+            - name: APPOPTICS_ENABLE_ELASTICSEARCH
+              value: 'false'
+            - name: APPOPTICS_ENABLE_KUBERNETES
+              value: 'false'
```

Comment:
This deployment manifest is specifically for the Kubernetes plugin. The README.md outlines instructions for using this file in a specific manner, where only the Kubernetes plugin is enabled. Changing all of these default values breaks that.

If we want to offer an example manifest with the available ENV vars exposed in "default" form we should create a separate file here in the project.

Reply:
APPOPTICS_ENABLE_KUBERNETES and APPOPTICS_DISABLE_HOSTAGENT should be true and everything else should be false.

Reply:
@cmrust I've updated  vars, thanks for the catch